qemu/tests/cfg/block_resize.cfg

```diff
@@ -27,6 +33,21 @@
                 blk_extra_params_stg = "wwn=0x5000123456789abc"
             Windows:
                 need_rescan = yes
+            variants:
+                - default_partition:
```

Comment:
Is this variant meaningful? I think it is redundant.

Reply:
There are different default partition type for Linux and windows, it is necessary